ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -717,6 +717,18 @@ StackView {
                 tokenName: footer.token.name
                 remainingTokens: footer.token.remainingTokens
                 tokenSource: footer.token.artworkSource
+                chainName: footer.token.chainName
+
+                feeText: root.feeText
+                feeErrorText: root.feeErrorText
+                isFeeLoading: root.isFeeLoading
+                accounts: root.accounts
+
+                onDeployFeesRequested: {
+                    root.deployFeesRequested(footer.token.chainId,
```

Comment:
I wonder which parameters are required to calculate fee for burn operation. Does the number of tokens matter? From what I understand, at least for ERC721 it should affect the final cost. So the question is if we pass all necessary parameters of the operation to calculate fee properly or maybe we are missing sth?

Reply:
Let me re-check!

Reply:
You were right. I was using the wrong backend call `computeDeployFee` instead of using `computeBurnFee`.
I've pushed changes and this will need an extra backend task for integration. So now, the burn action is still not taking into account the `accountAddress` neither for compute fee nor for burning.

Reply:
Here backend integration task: https://github.com/status-im/status-desktop/issues/11827 
CC: @endulab